antidox/perspective.py

```diff
@@ -76,26 +72,31 @@ def contains_pii(dlp_response):
       has_pii = True
   return has_pii
 
-# def contains_toxicity(perspective_response):
-#   if perspective_response['attributeScores']['TOXICITY']['summaryScore']['value'] >= .05
+#Checking and returning only for comments with a toxicity value of over 50 percent.
+def contains_toxicity(perspective_response):
+  is_toxic = False
+  if perspective_response['attributeScores']['TOXICITY']['summaryScore']['value'] >= .5:
+    print (perspective_response['attributeScores']['TOXICITY']['summaryScore']['value'])
```

Comment:
Can you remove this print statement, since you are printing the same value in the main method?